clang/include/clang/CIR/Dialect/IR/CIROpenCLAttrs.td

```diff
@@ -92,6 +92,27 @@ def OpenCLKernelMetadataAttr
 
 }
 
+//===----------------------------------------------------------------------===//
+// OpenCLVersionAttr
+//===----------------------------------------------------------------------===//
+
+def OpenCLVersionAttr : CIR_Attr<"OpenCLVersion", "cl.version"> {
+  let summary = "OpenCL version";
+  let parameters = (ins "int32_t":$major, "int32_t":$minor);
+  let description = [{
+    Represents the version of OpenCL.
+
+    Example:
+    ```
+    // Module compiled from OpenCL 1.2.
+    module attributes {cir.cl.version = cir.cl.version<1,2>} {}
```

Comment:
Super-Nit: The attribute is printed `cir.cl.version<1, 2>` (with a space after the comma) according to the added lit test.